front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/tables/csv.ts

```diff
@@ -119,14 +119,21 @@ async function handler(
           case "invalid_url":
           case "title_too_long":
           case "missing_csv":
-          case "invalid_csv":
             return apiError(req, res, {
               status_code: 400,
               api_error: {
                 type: "invalid_request_error",
                 message: upsertRes.error.message,
               },
             });
+          case "invalid_csv_content":
```

Comment:
handle invalid_csv_content specifically to return a code that connectors knows of signifying that the content is not valid